src/ansys/geometry/core/connection/conversions.py

```diff
@@ -576,3 +579,84 @@ def trimmed_curve_to_grpc_trimmed_curve(curve: "TrimmedCurve") -> GRPCTrimmedCur
         interval_start=i_start,
         interval_end=i_end,
     )
+
+
+def surface_to_grpc_surface(surface: Surface) -> tuple[GRPCSurface, GRPCSurfaceType]:
+    """Convert a ``Surface`` object to a surface gRPC message.
+
+    Parameters
+    ----------
+    surface : Surface
+        Surface to convert.
+
+    Returns
+    -------
+    GRPCSurface
+        Return ``Surface`` as a ``ansys.api.geometry.Surface`` message.
+    """
+    grpc_surface = None
+    surface_type = None
+    origin = point3d_to_grpc_point(surface.origin)
+    reference = unit_vector_to_grpc_direction(surface.dir_x)
+    axis = unit_vector_to_grpc_direction(surface.dir_z)
+
+    if isinstance(surface, Plane):
+        grpc_surface = GRPCSurface(origin=origin, reference=reference, axis=axis)
+        surface_type = GRPCSurfaceType.SURFACETYPE_PLANE
+    elif isinstance(surface, Sphere):
+        grpc_surface = GRPCSurface(
+            origin=origin, reference=reference, axis=axis, radius=surface.radius.m
+        )
+        surface_type = GRPCSurfaceType.SURFACETYPE_SPHERE
+    elif isinstance(surface, Cylinder):
+        grpc_surface = GRPCSurface(
+            origin=origin, reference=reference, axis=axis, radius=surface.radius.m
+        )
+        surface_type = GRPCSurfaceType.SURFACETYPE_CYLINDER
+    elif isinstance(surface, Cone):
+        grpc_surface = GRPCSurface(
+            origin=origin,
+            reference=reference,
+            axis=axis,
+            radius=surface.radius.m,
+            half_angle=surface.half_angle.m,
+        )
+        surface_type = GRPCSurfaceType.SURFACETYPE_CONE
+    elif isinstance(surface, Torus):
+        grpc_surface = GRPCSurface(
+            origin=origin,
+            reference=reference,
+            axis=axis,
+            major_radius=surface.major_radius.m,
+            minor_radius=surface.minor_radius.m,
+        )
+        surface_type = GRPCSurfaceType.SURFACETYPE_TORUS
+
+    return grpc_surface, surface_type
+
+
+def trimmed_surface_to_grpc_trimmed_surface(
+    trimmed_surface: "TrimmedSurface",
```

Comment:
```suggestion
    trimmed_surface: TrimmedSurface,
```
No need for he double quotes since the import statement is out of the type checking section.